docs/resource_assembly_migration.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Tiltfile Resource Assembly Migration
+layout: docs
+---
+
+Tilt is changing how it combines k8s objects and images into Tilt resources.
+
+If you don't want to deal with this right now, you can get back the old
+behavior for now by adding `k8s_resource_assembly_version(1)` to the top of
+your Tiltfile. We plan to remove that option on 2019-05-01. [Please let us know](http://localhost:4001/faq.html#q-how-do-i-get-help-with-tilt) if you have questions or concerns about any of this!
+
+The old behavior was roughly to make one Tilt resource per image, and associate
+a resource with all k8s objects that use that image. This led to confusing behavior
+when multiple Deployments used the same image, and was just generally unintuitive.
+
+The new behavior is to simply make one Tilt resource per k8s object that has a
+container. It's described [here](tiltfile_concepts.html#resources).
```

Comment:
nit: it's a little more accessible if hyperlinks are longer, and the text more accurately describes what they are.
```suggestion
container. ([Read more about the new behavior](tiltfile_concepts.html#resources)).
```